js/script.js

```diff
@@ -0,0 +1,165 @@
+const newPlayer = (name, symbol, turn) => {
+  const playerName = name;
+  const token = symbol;
+  const playerTurn = turn;
+  return { playerName, token, playerTurn };
+};
+
+const gameBoard = () => {
+  let board;
+
+  const init = () => {
+    board = [null, null, null, null, null, null, null, null, null];
+    document.getElementById('cell0').innerHTML = '';
+    document.getElementById('cell1').innerHTML = '';
+    document.getElementById('cell2').innerHTML = '';
+    document.getElementById('cell3').innerHTML = '';
+    document.getElementById('cell4').innerHTML = '';
+    document.getElementById('cell5').innerHTML = '';
+    document.getElementById('cell6').innerHTML = '';
+    document.getElementById('cell7').innerHTML = '';
+    document.getElementById('cell8').innerHTML = '';
+    document.getElementById('winner-text').innerHTML = '';
+  };
+
+  const setCell = (index, symbol) => {
+    board[index] = symbol;
+  };
+
+  const checkWinCondition = () => {
+    let isWin = false;
+    if (board[0] != null && board[0] === board[1] && board[1] === board[2]) isWin = true;
+    if (board[3] != null && board[3] === board[4] && board[4] === board[5]) isWin = true;
+    if (board[6] != null && board[6] === board[7] && board[7] === board[8]) isWin = true;
+    if (board[0] != null && board[0] === board[3] && board[3] === board[6]) isWin = true;
+    if (board[1] != null && board[1] === board[4] && board[4] === board[7]) isWin = true;
+    if (board[2] != null && board[2] === board[5] && board[5] === board[8]) isWin = true;
+    if (board[0] != null && board[0] === board[4] && board[4] === board[8]) isWin = true;
+    if (board[6] != null && board[6] === board[4] && board[4] === board[2]) isWin = true;
+
+    return isWin;
+  };
+
+  const checkDrawCondition = () => {
+    let isDraw = true;
+    if (board.includes(null)) isDraw = false;
+
+    return isDraw;
+  };
+
+  function blockCells() {
+    const cells = document.querySelectorAll('.cell');
+    [...cells].forEach(cell => { cell.style.pointerEvents = 'none'; });
+  }
+
+  return {
+    init,
+    setCell,
+    checkWinCondition,
+    checkDrawCondition,
+    blockCells,
+  };
+};
+
+const gameLogic = () => {
+  let player1;
+  let player2;
+  let c0;
+  let c1;
+  let c2;
+  let c3;
+  let c4;
+  let c5;
+  let c6;
+  let c7;
+  let c8;
```

Comment:
Find a way to use either only one variable or directly use the `board` from GameBoard.
Don't forget that you can make the `GameBoard's board` as an array of HTML tag.

Reply:
solved as for [this commit](https://github.com/firmoholanda/tictactoe-js/pull/4/commits/f9584425affac20b4b54a1684ff94c80da4784b9)